src/onefetch/info.rs

```diff
@@ -23,6 +25,8 @@ pub struct Info {
     repo_url: String,
     number_of_commits: String,
     lines_of_code: usize,
+    packed_repo_size: String,
+    files_count: Option<u128>,
```

Comment:
I'm curious why you decided to use `u128`. Will `u64::MAX` or `usize::MAX` to be insufficient?

Reply:
Because `u128` was already used for line counts. But you are right that u64 should be sufficient (any code base that would cause an u64 overflow deserves it ;) ).